### PR TITLE
Backport gdk4-win32 to 0.5

### PR DESCRIPTION
**Pull request by elmarco**
See #1103

### CODE REVIEW

File: gdk4-win32/Cargo.toml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+version = "0.5.0"
+
+[dependencies.gdk]
+path = "../gdk4"
+package = "gdk4"
+version = "0.5.0"
+
+[dependencies.gio]
+git = "https://github.com/gtk-rs/gtk-rs-core"
+version = "0.17"
```
Comment (bilelmoussaoui) [inline gdk4-win32/Cargo.toml:49]: should be set to 0.16 as well as adding a `branch = "0.16"` for all the gtk-rs-core dependencies

**Comment by bilelmoussaoui** [comment]
[This commit](https://github.com/gtk-rs/gtk4-rs/commit/aa2332f06bd6fbfae9a1b484b35e913184a28bc1) should be backported as well

**Comment by bilelmoussaoui** [comment]
https://github.com/gtk-rs/gtk4-rs/pull/1171 should be backported as well once it lands

**Comment by bilelmoussaoui** [comment]
https://github.com/gtk-rs/gtk4-rs/commit/b48d8cc4909039a39f21c64ea7f12424006080b3 is also needed to fix the windows msvc CI

**Comment by elmarco** [comment]
yay! gdk-win32 0.5 release coming? :)

**Comment by elmarco** [comment]
https://crates.io/crates/gdk4-win32 \o/

**Pull request merged**